src/psyclone/gocean0p1.py

```diff
@@ -289,6 +292,10 @@ def gen_code(self, parent):
                 self.stop_expr = Reference(
                     self.field_space + "%jstop", parent=self)
 
+        # Update our bounds in the PSyIR
+        self.children[0] = self.start_expr
```

Comment:
Isn't self.start_expr the same as self.children[0]? Similarly for self.stop_expr? If so, you don't need these lines.

Reply:
These lines are here because our bounds might have changed and therefore we need to correct the appropriate child nodes.

Reply:
As discussed, these are the same thing. I'll remove them rather than requesting a change.